libraries/WiFi/src/WiFiGeneric.h

```diff
@@ -174,6 +174,8 @@ class WiFiGenericClass
 
     bool initiateFTM(uint8_t frm_count=16, uint16_t burst_period=2, uint8_t channel=1, const uint8_t * mac=NULL);
 
+	bool setAntennaConfig(uint8_t gpio_ant1, uint8_t gpio_ant2, uint8_t default_rx_ant);
```

Comment:
indentation here seems wrong :)

Reply:
I would suggest that you rename the method to `setDualAntennaConfig` to make it more clearer that it should be used only when dual antenna is available.